test/integration/clinicians/clinician-sidebar.js

```diff
@@ -449,9 +459,9 @@ context('clinician sidebar', function() {
   specify('view clinician', function() {
```

Comment:
Don't think this test does anything unique? I think the same things are done in the `edit clinician` test.

Reply:
yeah if we can remove this without coverage drop, we should.  edit was probably added later, duplicating this when it should have just been modified

Reply:
This was removed and test coverage remained at 100% 👍